### DISCUSSION TITLE
Multi languages

### DISCUSSION THREAD

**Discussion by aharishsundhar** [GrapesJS/grapesjs#5318] [Q&A]
@artf if your available i have one doubt if it's available for blocks change in multi languages html component add on it or it available mult languages plugins please mention it here

**Comment by lexoyo** [comment]
I'm not sure to understand but I think you're asking how to create multilingual blocks?

When you create a new plugin with grapesjs cli, it asks you if you want to create blocks and if you needd i18n. Just say yes to both and you will get the files and code for that

**Reply by aharishsundhar** [reply]
yes right
